travis/test_server.py

```diff
@@ -390,7 +393,7 @@ def main(argv=None):
             test_loghandler = ['openerp.tools.yaml_import:DEBUG',
                                'openerp.models.schema:DEBUG']
     odoo_full = os.environ.get("ODOO_REPO", "odoo/odoo")
-    server_path = get_server_path(odoo_full, odoo_version, travis_home)
+    server_path = get_server_path(odoo_full, odoo_version, odoo_branch, travis_home)
```

Comment:
I fixed as follows 
```python
server_path = get_server_path(odoo_full, odoo_branch or odoo_version, travis_home)
```
I did not know that you could use an or in the parameters, this is amaizing